pkg/nextgen/distdoc/dotviz_test.go

```diff
@@ -0,0 +1,69 @@
+package distdoc
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// DotGraphVisitor is a graph visitor that outputs the walked path as "dot" diagram.
+type DotGraphVisitor struct {
```

Comment:
Interesting file, no tests. More a helper utility for debugging?

Reply:
Yes it output the DAG as dot-file so it can be rendered to an image (with nodes and arrows). I used it for debugging and I want to keep because I foresee it will become in handy analysing how the DAG is processed in the feature. Maybe even for production networks using the CLI;

`nuts network render-dag-graph`

or something.

But since we're not there yet I marked it as a test file.